nitrite_test.go

```diff
@@ -0,0 +1,108 @@
+package nitrite_test
+
+import (
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/fxamacker/cbor/v2"
+	"github.com/hf/nitrite"
+)
+
+func requireNoError(t *testing.T, got error) {
+	if got != nil {
+		t.Fatalf("unexpected error: %v", got)
+	}
+}
+
+func requireEqual(t *testing.T, got, want interface{}) {
+	if got != want {
+		t.Fatalf("not equal: got %v, want %v", got, want)
+	}
+}
+
+func requireErrorIs(t *testing.T, got, want error) {
+	if !errors.Is(got, want) {
+		t.Fatalf("unexpected error type: got %T, want %T", got, want)
+	}
+}
+
+type testingCOSEPayload struct {
+	_ struct{} `cbor:",toarray"`
+
+	Protected   []byte
+	Unprotected cbor.RawMessage
+	Payload     []byte
+	Signature   []byte
+}
+
+func TestAttestationCreatedAt(t *testing.T) {
+	timeToMillis := func(t time.Time) uint64 {
+		return uint64(t.UnixNano() / 1e6)
+	}
```

Comment:
In go1.17 we could use `time.UnixMilli`, but I didn't want to bump the go version